com.unity.formats.fbx/Editor/ExportSettings.cs

```diff
@@ -762,7 +762,8 @@ private static HashSet<string> GetDefaultVendorLocations()
                 return existingDirectories;
             }
 
-            throw new NotImplementedException();
+            // empty list for not implemented (e.g. Linux)
+            return new HashSet<string>();
```

Comment:
I believe it's `/usr/autodesk` on linux.

We threw the exception on purpose to force us to figure it out when we ported to another platform.

Reply:
not sure how to check this works unless I have a linux machine I can install Maya onto

Reply:
Let's add /usr/autodesk for linux, we can fix it later if needed.

Ideally the code would be:
* set `defaults` based on the platform
* do the loop filtering for the directory existing on all platforms the same way